ShadowDOM/explainer.md

```diff
@@ -243,6 +245,20 @@ Another advantage of this proposal is that it can allow multiple module specifie
 </my-element>
 ```
 
+### Scoping
+
+The module map exists today as a global registry per document, not scoped to a particular shadow root. Many developers have expressed interest in such a global map for sharing stylesheets, as it allows for nested shadow roots to access a base set of shared styles without needing to redefine them at each level of shadow root nesting.
+
+A global map does come with some tradeoffs, particularly when names collide. With a global map, nested shadow roots could override entries from parent shadow roots, which could be undesirable.
+
+### `<script>` vs `<style>` For CSS Modules
+
+This document uses the `<script>` tag for defining CSS modules. Developer feedback has shown a preference for using the `<style>` tag for CSS Modules. This makes sense for CSS Modules in isolation, but does not align with other types of modules. The table in the [next section](#other-declarative-modules) details other module types and demonstrates the degree of consistency achieved with `<script>`.  Developer feedback is important and should be considered, even at the potential expense of consistency.
```

Comment:
"using the `<style>` tag for CSS Modules" -> "using the `<style>` tag when defining a CSS Module"